File: data/java/COUNT_PAIRS_WHOSE_PRODUCTS_EXIST_IN_ARRAY_TRANSLATED.py
====================
def count_pairs_whose_products_exist_in_array ( arr , n ) :
    result = 0
    for i in range ( n ) :
        for j in range ( i + 1 , n ) :
            product = arr [ i ] * arr [ j ]
            for k in range ( n ) :
                if arr [ k ] == product :
                    result += 1
                    break
    return result

File: data/python/COUNT_PAIRS_WHOSE_PRODUCTS_EXIST_IN_ARRAY_TRANSLATED.java
====================
public static int f_gold ( int [ ] arr , int n ) {
  int result = 0 ;
  ;
  for ( int i = 0 ;
  i <= n ;
  i ++ ) {
    for ( int j = i + 1 ;
    j <= n ;
    j ++ ) {
      int product = arr [ i ] * arr [ j ] ;
      ;
      for ( int k = 0 ;
      k <= n ;
      k ++ ) {
        if ( ( arr [ k ] == product ) && ( arr [ k ] != 0 ) ) {
          result = result + 1 ;
          break ;
        }
      }
    }
  }
  return result ;
}

